src/libraries/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CapiHelper.DSA.Shared.cs

```diff
@@ -178,7 +178,6 @@ internal static DSAParameters ToDSAParameters(this byte[] cspBlob, bool includeP
                         //  BYTE[lenP]      Y
                         //  BYTE[lenX]      X (if private)
 
-                        int magic = br.ReadInt32(); // Expected to be DSS_PUB_MAGIC_VER3 or DSS_PRIV_MAGIC_VER3
```

Comment:
All of these deleted calls to Read* methods have side effects.  The results can be ignored, but the calls can't be removed.

Hopefully lots of tests explode :smile:.

Reply:
Do we prefer `_ = br.ReadInt32();` or just `br.ReadInt32()` ? The latter might be flagged by analyzers, but I assume not as it's obviously side effecting.

Reply:
I'd probably do it as

```C#
// skip past dwMagic (Expected to be DSS_PUB_MAGIC_VER3 or DSS_PRIV_MAGIC_VER3)
br.ReadInt32();
```

Unless an analyzer warned, then I'd use the discard-assignment.

Reply:
Thank you for your review!
i updated it you said

Reply:
also,  it's not flagged by analyzers in VS2019